eng/pipelines/libraries/build-job.yml

```diff
@@ -70,13 +70,8 @@ jobs:
         - ${{ parameters.variables }}
 
       steps:
-        - ${{ if eq(parameters.osGroup, 'OSX') }}:
-          - script: |
-              brew install pkgconfig icu4c openssl
-              brew link --force icu4c
-              ln -s /usr/local/opt/openssl/lib/pkgconfig/libcrypto.pc /usr/local/lib/pkgconfig/
-              ln -s /usr/local/opt/openssl/lib/pkgconfig/libssl.pc /usr/local/lib/pkgconfig/
-              ln -s /usr/local/opt/openssl/lib/pkgconfig/openssl.pc /usr/local/lib/pkgconfig/
+        - ${{ if in(parameters.osGroup, 'OSX', 'iOS', 'tvOS') }}:
```

Comment:
Merge with the if statement for checking disk usage?

Reply:
Done in Done in 43a3d87b96323df90d67326f252eb632c56611ab.